app/helpers/dateable.rb

```diff
@@ -16,4 +16,19 @@ def date_range
 
     start_date..end_date
   end
+  
+  def date_range_picker
+    start_date = date_params[:date_from]&.split('/')
+    end_date = date_params[:date_to]&.split('/')
+
+    start_date = start_date ? "#{start_date[1]}/#{start_date[0]}/#{start_date[2]}"&.to_date : "Jan, 2, 1970".to_date
```

Comment:
Was there something wrong with the date_range function above/ do we need to deconstruct the dates in this way for some reason?

Reply:
Yes, because before of the changes the front-end sent the date in format: 
YYYY-MM-DD, and in this format the rails/ruby was able parse 'string' to 'date object', but now, with the gem, the format of front-end send is 'MM/DD/YYYY', and i did a few tests and the rails/ruby could not parse to date, and 'DD/MM/YYYY' is able to parse.

Reply:
Aha! In that case, I think you could make an explicit call like `DateTime.parse(start_date)` and it should allow it to work again without doing any manual parsing :)

Reply:
You *might* also be able to provide some JS configuration options to the datepicker so that it renders the date in the format that's expected

Either way would work though. The less data mutation we have to do, the better

Reply:
Guys, I think than I resolved the problem... Well, I created a verification inside a function `date_range` and now it's working. I couldn't did just `DateTime.parse(...)` because the format MM/DD/YYYY was not accepted. But with `Date.strptime` is working!